src/utils/signInHistory.ts

```diff
@@ -118,14 +118,18 @@ export const formatData = (
 
     if (!newRow.eventType) continue;
 
-    newRow.time = prettifyDate(Number(row["timestamp"]), {
-      month: "long",
-      day: "numeric",
-      year: "numeric",
-      hour: "numeric",
-      minute: "numeric",
-      hourCycle: "h12",
-      timeZone: "GB",
+    newRow.time = prettifyDate({
+      dateEpoch: Number(row["timestamp"]), 
+      options: {
+        month: "long",
+        day: "numeric",
+        year: "numeric",
+        hour: "numeric",
+        minute: "numeric",
+        hourCycle: "h12",
+        timeZone: "GB",
+      },
+      locale: currentLanguage
```

Comment:
Should we default to `en-gb` if the language is valid, but a non supported language (e.g. `fr-fr`)

Reply:
The way I handled that scenario (or attempted to) in `prettifyDate` is by checking whether the code that's been passed through is a key in the list of `LANGUAGE_CODES` that I added (currently we only support `cy` and `en` translations so I mapped those two to their corresponding language tags) and falling back to `en-GB` if it doesn't match.

Not sure whether my approach to the problem was the the most optimal one but if you have any feedback, I'm open to suggestions!